`ManyToManyField.through` no longer has `_default_manager` attribute

# Bug report

Following the call for testing in #1740, I tried `django-stubs` git master and ran into this issue.

For example if `MyModel` has relation `manytomany = models.ManyToManyField(OtherModel)`

## What's wrong

When accessing `MyModel.manytomany.through._default_manager`, I'm getting error:

> "type[MyModel_manytomany]" has no attribute "_default_manager"  [attr-defined]

`MyModel.manytomany.through.objects` works fine, however.

I can also reproduce this after these changes in django-stubs test suite:

```patch
diff --git a/tests/typecheck/fields/test_related.yml b/tests/typecheck/fields/test_related.yml
index 8f21557e..8d0ce289 100644
--- a/tests/typecheck/fields/test_related.yml
+++ b/tests/typecheck/fields/test_related.yml
@@ -337,6 +337,7 @@
         from myapp.models import User
         reveal_type(User().friends)  # N: Revealed type is "django.db.models.fields.related_descriptors.ManyRelatedManager[myapp.models.User]"
         reveal_type(User.friends.through.objects.get())  # N: Revealed type is "myapp.models.User_friends"
+        reveal_type(User.friends.through._default_manager)
         reveal_type(User.friends.through().from_user)  # N: Revealed type is "myapp.models.User"
         reveal_type(User.friends.through().from_user_id)  # N: Revealed type is "builtins.int"
         reveal_type(User.friends.through().to_user)  # N: Revealed type is "myapp.models.User"
```

```
E   pytest_mypy_plugins.utils.TypecheckAssertionError: Invalid output: 
E   Actual:
E     ...
E     main:4: error: "Type[User_friends]" has no attribute "_default_manager" (diff)
```

## System information

- OS: macOS 13.6
- `python` version: 3.11.5
- `django` version: 4.1.11
- `mypy` version: 1.5.1
- `django-stubs` version: (git master, commit 84a085c784ee68318d6fb190cbdda216d569fb43)
- `django-stubs-ext` version: 4.2.2

--- a/mypy_django_plugin/transformers/models.py
+++ b/mypy_django_plugin/transformers/models.py
@@ -740,6 +740,12 @@ class ProcessManyToManyFields(ModelClassInitializer):
                     name="objects",
                     sym_type=Instance(manager_info, [Instance(through_model, [])]),
                 )
+                # Also add manager as '_default_manager' attribute
+                helpers.add_new_sym_for_info(
+                    through_model,
+                    name="_default_manager",
+                    sym_type=Instance(manager_info, [Instance(through_model, [])]),
+                )
 
     @cached_property
     def default_pk_instance(self) -> Instance:

--- a/tests/typecheck/fields/test_related.yml
+++ b/tests/typecheck/fields/test_related.yml
@@ -337,6 +337,7 @@
         from myapp.models import User
         reveal_type(User().friends)  # N: Revealed type is "django.db.models.fields.related_descriptors.ManyRelatedManager[myapp.models.User]"
         reveal_type(User.friends.through.objects.get())  # N: Revealed type is "myapp.models.User_friends"
+        reveal_type(User.friends.through._default_manager)  # N: Revealed type is "django.db.models.manager.Manager[myapp.models.User_friends]"
         reveal_type(User.friends.through().from_user)  # N: Revealed type is "myapp.models.User"
         reveal_type(User.friends.through().from_user_id)  # N: Revealed type is "builtins.int"
         reveal_type(User.friends.through().to_user)  # N: Revealed type is "myapp.models.User"
@@ -1088,6 +1089,7 @@
 
         reveal_type(MyModel.auto_through.through)
         reveal_type(MyModel.auto_through.through.mymodel)
+        reveal_type(MyModel.auto_through.through._default_manager)
 
         reveal_type(MyModel.other_again.through)
     out: |
@@ -1106,7 +1108,8 @@
         main:17: note: Revealed type is "builtins.int"
         main:19: note: Revealed type is "Type[myapp.models.MyModel_auto_through]"
         main:20: note: Revealed type is "django.db.models.fields.related_descriptors.ForwardManyToOneDescriptor[django.db.models.fields.related.ForeignKey[Union[myapp.models.MyModel, django.db.models.expressions.Combinable], myapp.models.MyModel]]"
-        main:22: note: Revealed type is "Type[myapp.models.MyModel_other_again]"
+        main:21: note: Revealed type is "django.db.models.manager.Manager[myapp.models.MyModel_auto_through]"
+        main:23: note: Revealed type is "Type[myapp.models.MyModel_other_again]"
     installed_apps:
         -   myapp
     files:

